### PR TITLE
Merge branch 'feature/nuget' into 'master'

### PR DESCRIPTION
**Pull request by RaymondLuong3**
There were no merge conflicts.
Please review https://github.com/sillsdev/libpalaso/pull/1035 which updates the reference to CHANGELOG.md and GitVersion.yml before reviewing this.

This change is [Reviewable](https://reviewable.io/reviews/sillsdev/libpalaso/1036)

### CODE REVIEW

File: ReadmeDesignerProjects.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# WinForms designer
+
+Starting with .NET Core 3.1 it's now possible to use WinForms in sdk-style
+csproj files. However, support for opening files in Designer is still not
```
Comment (tombogle) [inline ReadmeDesignerProjects.md:4]: I think this is no longer true. At least that's what we're attempting to prove.

**Review by tombogle — CHANGES_REQUESTED** [review]

File: ReadmeDesignerProjects.md (hunk 1 above)
Comment (RaymondLuong3) [reply ReadmeDesignerProjects.md:4]: Ok, that is true. I'm unaware of designer projects in libpalaso but they can very well be there. I'll leave this file in here, and will update the comment to what we know will be accurate.

**Comment by tombogle** [comment]
___
*[ReadmeDesignerProjects.md, line 4 at r1](https://reviewable.io/reviews/sillsdev/libpalaso/1036#-MO8uoZnRU7HHNntgiIr-r1-4:-MOJ0tg-6NcHRlr_Ed7j:b-n77ulx) ([raw file](https://github.com/sillsdev/libpalaso/blob/7c90a37d718ef4c0e09991555d0b21d3a7bc8e2b/ReadmeDesignerProjects.md#L4)):*
> part of 
Yes, there are lots of Designer files in libpalaso.

**Comment by RaymondLuong3** [comment]
@tombogle Did you see anything else that needed adjusted for this merge? I didn't quite understand from your last comment.

File: ReadmeDesignerProjects.md (hunk 1 above)
Comment (ermshiperete) [reply ReadmeDesignerProjects.md]: I think we should delete that file. It's outdated and the content is only partially true nowadays (even with the edit).

Comment (RaymondLuong3) [reply ReadmeDesignerProjects.md]: Ok, I removed the readme.

**Review by RaymondLuong3 — COMMENTED** [review]
I had to adjust team city in order to get builds to pass. For the windows builds, I set configuration > Debug (was previously DebugStrongName) and changed the artifacts paths. I also updated the build tool to be MS Build 16.0 and VS 2019. On the win-master-continuous agent I removed the /p:Platform=x64 so it would use the default (AnyCPU). On the Linux build the configuration and artifacts paths are Debug (from DebugMonoStrongName).

> *[Reviewable](https://reviewable.io/reviews/sillsdev/libpalaso/1036#-:-MUG9JWt3ONxGrcvwmPK:btwwv0b)* status: 0 of 496 files reviewed, 1 unresolved discussion (waiting on @tombogle)

**Pull request merged**